README.md

```diff
@@ -1,24 +1,203 @@
-# README
+<div align="center">
+ 
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+  <h3><b><a name="Recipe-app"></a></b></h3>
 
-Things you may want to cover:
+</div>
 
-* Ruby version
+<!-- TABLE OF CONTENTS -->
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+    - [🚀 Video Presentation](#presentation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 [Budget App] <a name="about-project"></a>
 
-* How to run the test suite
+> 
 
-* Services (job queues, cache servers, search engines, etc.)
+**[Budget App]** Budget app is an app for managing transactions with categorized lists, sign-up and login pages, and navigation buttons for moving between pages. Users can add new categories or transactions.
 
-* Deployment instructions
 
-* ...
+>based on the ERD diagram
+<img src ="./budget_erd.png">
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add Type of transaction]**
+- **[Show your transactions for given category]**
+- **[Give total of each transaction]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-n0m8.onrender.com)- Powered by Render
+
+## 🚀 Video Presentation <a name="presentation"></a>
```

Comment:
- [ ] during your presentation i did not see any part you talked about the most interesting part of your code or a code you are proud of, kindly add that to your video